### PR TITLE
Begin recipe class

### PR DESCRIPTION
**Pull request by cagallo**
#### What's this PR do?

- Begin recipe class and test

#### Where should the reviewer start?

- recipe.js

#### How should this be manually tested?

- N/A

#### Any background context you want to provide?

- Began this recipe class, we might need an Ingredients class

#### What are the relevant tickets?

- N/A

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A

### CODE REVIEW

File: src/classes/Recipe.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Recipe {
```
Comment (anewb87) [inline src/classes/Recipe.js:1]: Here we go!

**Review by kammon10 — APPROVED** [review]
great start!

File: test/Recipe-test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai';
```
Comment (anewb87) [inline test/Recipe-test.js:1]: TDD- going to start here as we keep going. Did want to put in the file for the class and do the bare bones outline.

File: test/RecipeRepository-test.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import recipeData from '../src/data/recipes';
 import ingredientsData from '../src/data/ingredients';
 
 
-describe('Recipe', () => {
+describe('RecipeRepository', () => {
```
Comment (aliroemhildt) [inline test/RecipeRepository-test.js:7]: I agree, this makes more sense

**Review by aliroemhildt — APPROVED** [review]

**Review by anewb87 — APPROVED** [review]
Skeleton to start.

**Pull request merged**